### ISSUE TITLE
Using SystemJS for AMD

### ISSUE THREAD

**Issue by hirokiosame** [systemjs/systemjs#1870]
I'm having trouble doing something as simple as AMD defining a file and invoking it with SystemJS 2.0.2:
```
<script type="systemjs-packagemap">
{
  "packages": {
    "vue": "//unpkg.com/vue@2.5.17/dist/vue.min.js"
  }
}
</script>
```

```
define('demo', ['vue'], function (Vue) {
	console.log(Vue);
});

System.import('demo');
```

Here's the fiddle:
http://jsfiddle.net/hirokiosame/wd7L3cp2/

Any direction to making this work would be appreciated.

Also, I understand package maps are the only thing implemented in the configuration so far. Will there be support for `default extension` or a way to add it with the current version?

**Comment by guybedford** [comment]
I've implemented a fix to the bundle scheme in 2.1.0 so that this example should work now when combined with the named register extension being loaded after SystemJS - https://github.com/systemjs/systemjs/blob/master/dist/extras/named-register.js.

Let me know if that works for you here.

**Comment by guybedford** [comment]
This should be fixed in 2.1.1 when using both the AMD extension and the named register extension.

**Comment by hirokiosame** [comment]
Thank you @guybedford ! Works now :)
